guake/guake_app.py

```diff
@@ -1377,8 +1404,8 @@ def restore_tabs(self, filename='session.json', suppress_notify=False):
                             directory, tab['label'], tab['custom_label_set']
                         )
                         if tab.get('panes', False):
-                            if directory:
-                                continue
+                            # if directory:
```

Comment:
@mlouielu 
With these lines not all tabs were restored.
But perhaps there was a reason to add them. Is it safe to remove these lines?